rust/src/measurements/rappor/mod.rs

```diff
@@ -0,0 +1,111 @@
+use crate::error::Fallible;
+use crate::metrics::DiscreteDistance;
+use crate::measures::MaxDivergence;
+use crate::domains::{AtomDomain, VectorDomain};
+use crate::core::{Function, Measurement, PrivacyMap};
+use crate::traits::{InfSub, InfDiv, InfLn, InfMul, samplers::SampleBernoulli};
+
+
+/// Make a Measurement that implements Basic RAPPOR
```

Comment:
Should `#[bootstrap(...)]` be added here, unless there's a reason not to generate Python and R bindings? Or maybe it would be good, but it could be a follow-up issue?

Reply:
Would love to have this, just wasn't sure how to go about it. Will check the docs and have a try at making it work soon

Reply:
I was going to handle this part myself!